gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -74,7 +74,7 @@ public class ConfigurationKeys {
   public static final boolean DEFAULT_STATE_STORE_COMPRESSED_VALUES = true;
   // DB state store configuration
   public static final String STATE_STORE_DB_JDBC_DRIVER_KEY = "state.store.db.jdbc.driver";
-  public static final String DEFAULT_STATE_STORE_DB_JDBC_DRIVER = "com.mysql.jdbc.Driver";
```

Comment:
Is this branch not built off master? The diffs seem off since this was committed by you in a prior PR

Reply:
sorry those may have landed while this was under development... now fixed